test/unit/connection_string.test.ts

```diff
@@ -912,13 +912,11 @@ describe('Connection String', function () {
     });
 
     context('when option is invalid', function () {
-      it('it defaults to stderr', function () {
-        const client = new MongoClient('mongodb://a/?mongodbLogPath=stdnothing', {
-          [loggerFeatureFlag]: true
-        });
-        const log: Log = { t: new Date(), c: 'ConnectionStringInvalidOption', s: 'error' };
-        client.options.mongoLoggerOptions.logDestination.write(log);
-        expect(stderrStub.write).calledWith(inspect(log, { breakLength: Infinity, compact: true }));
+      it('should throw error at construction', function () {
+        expect(
+          () =>
+            new MongoClient('mongodb://a/?mongodbLogPath=stdnothing', { [loggerFeatureFlag]: true })
+        ).to.throw(MongoAPIError);
```

Comment:
Should we add more test cases here to check all edge cases for mongodbLogPath and mongodbLogComponentSeverities?

Reply:
Which edge cases would you like to check?

I should have included this in my PR description, but if you expand this test file and the `mongo_client.test.js` test file, you can view some other test cases for the client options added in as part of the original logging client options implementation in [NODE-4849](https://jira.mongodb.org/browse/NODE-4849) and [NODE-4878](https://jira.mongodb.org/browse/NODE-4878).

Reply:
This comment is not relevant anymore after you clarified the test cases in the kick-off doc. Thx for the detailed answers, helpful to gather the full context of the problem!